.jsdoc.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
     includePattern: '\\.js$'
   },
   templates: {
-    copyright: 'Copyright 2019 Google, LLC.',
+    copyright: 'Copyright 2018 Google, LLC.',
```

Comment:
Can we update the year?

Reply:
This file gets copied in with the other templates, so it would be kinda hard to change this.  I think the copyright is supposed to be the year the code was written, so this is probably fine.

Reply:
ok